doc/conf.py

```diff
@@ -326,5 +326,10 @@ def setup(app):
     'examples_dirs': examples_dirs,
     'gallery_dirs': gallery_dirs,
     'find_mayavi_figures': find_mayavi_figures,
-    'expected_failing_examples': ['../examples/plot_raise.py', '../examples/plot_syntaxerror.py']
+    'expected_failing_examples': ['../examples/plot_raise.py', '../examples/plot_syntaxerror.py'],
+    'binder_org': 'choldgraf',
```

Comment:
FWIW, the binder thing will still work, as long as the site exists somewhere on gh-pages. So as a stopgap measure we can still just push the site documentation to gh-pages even if it's not being served by github as the website.